gcp_variant_transforms/libs/annotation/vep/vep_runner_util.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
```

Comment:
In general, please avoid creating generic util classes as much as possible since they become a bag of unrelated stuff over time (i'd argue that it already is). I think a better approach is to make a module that just contains the file distribution logic and then move the rest into vep_runner module (or create another module). I'm ok to leave as is for now though as I don't want to block this PR much longer :)